CONTRIBUTING.md

```diff
@@ -145,7 +145,9 @@ General guidelines for making a good gallery plot:
   plot the data*.
 * Describe the feature that you're showcasing and link to other relevant parts of the
   documentation.
-* SI units should be used in the example code for gallery plots.
+* SI units should be used in the example code for gallery plots and should follow the
+  [NIST unit rules and style conventions](https://physics.nist.gov/cuu/Units/checklist.html).
+  (e.g. add a space between the numerical value and the unit symbol).
```

Comment:
I think the period is not needed because there is another sentence in the parenthesis.
```suggestion
* SI units should be used in the example code for gallery plots and should follow the
  [NIST unit rules and style conventions](https://physics.nist.gov/cuu/Units/checklist.html)
  (e.g. add a space between the numerical value and the unit symbol).
```

Reply:
I thought that the "SI units should be used in the example code for gallery plots." statement was added so that we use e.g. `projection="x5c"` instead of `projection=`x5i`? The 'adding a space' convention seems to be relevant only for in-line comments or axis titles.

To be honest, this is actually the first time I've heard of NIST (I only knew that putting a space between the number and unit was a thing), so I'm wondering if it's actually common in the Geosciences? And what about units with superscripts (e.g. m^3 for volume), how do we want it to be formatted? Sorry for all the questions, I'm just wondering if we need to actually spell things out in so much detail.

Reply:
> I'm just wondering if we need to actually spell things out in so much detail.

I agree. It's good to have consistent styles in the documentation, but I'm not sure if we should have strict rules since it makes the contributing guides even longer.

Reply:
This is also the first time I know NIST.

Reply:
> 
> 
> I thought that the "SI units should be used in the example code for gallery plots." statement was added so that we use e.g. `projection="x5c"` instead of `projection=`x5i`? The 'adding a space' convention seems to be relevant only for in-line comments or axis titles.
> 

Correct @weiji14.

> To be honest, this is actually the first time I've heard of NIST (I only knew that putting a space between the number and unit was a thing), so I'm wondering if it's actually common in the Geosciences? And what about units with superscripts (e.g. m^3 for volume), how do we want it to be formatted? Sorry for all the questions, I'm just wondering if we need to actually spell things out in so much detail.

First of all, I'm sorry for raising all these questions. I learned all these conventions from the beginning of my studies, however, we can leave these points as is since I'm not sure if they are only recommendations or a scientific rule 😉.